apollo-compiler/src/main/kotlin/com/apollostack/compiler/TypeSpecBuilder.kt

```diff
@@ -0,0 +1,8 @@
+package com.apollostack.compiler
+
+import com.squareup.javapoet.TypeSpec
+
+interface TypeSpecBuilder {
```

Comment:
Not sure do we need this interface, as we can still use `CodeGenerator` and `packageName` used only in `GraphQLCompiler` where it even set